src/validation/index.js

```diff
@@ -0,0 +1,29 @@
+import helpers from '../utils/helpers.utils';
+import { numberRegex } from '../utils/regexen.utils';
+
+const { checkForEmptyFields, checkPatternedFields } = helpers;
+
+export default {
+  oneWayRequest: (req, res, next) => {
```

Comment:
This middleware is called on all POST `/requests` requests. I believe this will apply to other request types besides _"one-way"_ and as such should contain a logic to check if a request is of type _"one-way"_ before excluding the _"return date"_. Also, the function should be more generic than specific

Reply:
emalRegex and passWords, just ignore them. I wonder how they slipped past, I might probably remove them now or leave them to be used for later i.e. if anybody is doing RAW regular expression validation. I'll just remove it.

The POST route will default to one-way trip by default. Users on multi-city and return-trip will need to add a category query e.g. `http://localhost:3000/requests?category=multi-city`

Reply:
Please remove them emailRegex and passwordRegex, they would have been needed for the auth routes which have been taken care of. Otherwise they'll be unused code that will drop test coverage

Reply:
Thank and noted. But i repeat ```emalRegex and passWords, just ignore them. **I wonder how they slipped past,** I might probably remove them now or leave them to be used for later i.e. if anybody is doing RAW regular expression validation. I'll just remove it.```